webapp-django/crashstats/settings/base.py

```diff
@@ -291,7 +291,6 @@ def filter(self, record):
 PROCESS_TYPES = (
     "any",
     "parent",
-    "browser",
```

Comment:
Taking this out here means that it won't show up as a filter option in the TopCrashers report. That should reduce some confusion for people using that report.

I'm pretty sure that's the only place this list shows up, but Socorro is an antediluvian place of malevolence and horror.